### PR TITLE
feat: Draft PR For [FRONTEND] Form page (desktop) task

### PR DESCRIPTION
**Pull request by DanielEmmanuel1**
Form Page Rework (Desktop Version)

Changes made:

- Redesigned form page layout according to Figma designs for desktop view
- Enhanced UI components:
- Implemented new multiplier selector with dynamic gradient slider
- Updated token selector styling
- Improved balance cards display
- Refined form input fields and styling
- Maintained backend integration and functionality
- Preserved all existing form submission logic

### CODE REVIEW

File: frontend/src/components/BalanceCards.jsx
Comment (whateverfw) [inline frontend/src/components/BalanceCards.jsx:1]: balance card styling doesn't match figma design

File: frontend/src/components/MultiplierSelector.jsx
Comment (whateverfw) [inline frontend/src/components/MultiplierSelector.jsx:1]: multiplier selector styles don't match figma

File: frontend/src/components/TokenSelector.jsx
Comment (whateverfw) [inline frontend/src/components/TokenSelector.jsx:1]: token selector' styles don't match figma

File: frontend/src/components/BalanceCards.jsx [hunk 1]
```diff
@@ -1,17 +1,14 @@
...
-import { ReactComponent as DAI } from '../assets/icons/dai.svg';
 import { useMatchMedia } from 'hooks/useMatchMedia';
 import { getBalances } from '../services/wallet';
 
 const BalanceCards = ({ walletId }) => {
   const [balances, setBalances] = useState([
     { icon: <ETH />, title: 'ETH', balance: '0.00' },
-    { icon: <USDC />, title: 'USDC', balance: '0.00' },
-    { icon: <STRK />, title: 'STRK', balance: '0.00' },
-    { icon: <DAI />, title: 'DAI', balance: '0.00' },
```
Comment (whateverfw) [inline frontend/src/components/BalanceCards.jsx:14]: keep previous icons

File: frontend/src/pages/forms/Form.jsx
Comment (whateverfw) [inline frontend/src/pages/forms/Form.jsx:1]: page background doesn't match figma

**Review by whateverfw — CHANGES_REQUESTED** [review]
Please make sure that every part of the page is done according to figma, currently most of the styles don't match Figma

File: frontend/src/pages/forms/Form.jsx
Comment (whateverfw) [reply frontend/src/pages/forms/Form.jsx:1]: still unresolved

File: frontend/src/components/TokenSelector.jsx
Comment (whateverfw) [reply frontend/src/components/TokenSelector.jsx:1]: sizes still doesn't match figma

File: frontend/src/components/MultiplierSelector.jsx
Comment (whateverfw) [reply frontend/src/components/MultiplierSelector.jsx:1]: still not matching figma

File: frontend/src/components/BalanceCards.jsx
Comment (whateverfw) [reply frontend/src/components/BalanceCards.jsx:1]: still not matching figma

File: frontend/src/components/TokenSelector.jsx [hunk 2]
```diff
@@ -11,25 +11,32 @@ const Tokens = [
...
-          <h5>
-            {token.component} {token.label}
-          </h5>
-        </label>
-      </div>
-    ))}
-  </div>
-);
+const TokenSelector = ({ setSelectedToken }) => {
+  const [tokenSelect, setTokenSelect] = useState('STRK');
```
Comment (whateverfw) [inline frontend/src/components/TokenSelector.jsx:15]: why do we have this state? we already have `selectedToken`

**Review by whateverfw — CHANGES_REQUESTED** [review]

File: frontend/src/pages/forms/Form.jsx
Comment (DanielEmmanuel1) [reply frontend/src/pages/forms/Form.jsx:1]: I literally got the background.png used in dashboard component, please can u point me to the specific background I should be using.
Ty

File: frontend/src/components/TokenSelector.jsx (hunk 2 above)
Comment (whateverfw) [reply frontend/src/components/TokenSelector.jsx:15]: but we already have `selectedToken` in `Form.jsx` that can be passed to `TokenSelector` component

File: frontend/src/pages/forms/Form.jsx
Comment (whateverfw) [reply frontend/src/pages/forms/Form.jsx:1]: I don't think it looks the same
image
image

**Comment by djeck1432** [comment]
@DanielEmmanuel1 Any updates?

**Comment by DanielEmmanuel1** [comment]
> @DanielEmmanuel1 Any updates?

Yeah, I'm currently working on it right now

**Comment by DanielEmmanuel1** [comment]
@whateverfw Can u please go through it to confirm if its okay now

File: frontend/src/components/TokenSelector.jsx
Comment (whateverfw) [reply frontend/src/components/TokenSelector.jsx:1]: still doesn't match figma

File: frontend/src/pages/forms/Form.jsx
Comment (whateverfw) [reply frontend/src/pages/forms/Form.jsx:1]: overall now it is at least good direction, but you can clearly see that there are double stars in app, you should remove it.

File: frontend/src/components/TokenSelector.jsx (hunk 2 above)
Comment (whateverfw) [reply frontend/src/components/TokenSelector.jsx:15]: still unresolved after latest changes

**Comment by whateverfw** [comment]
1) Background is more or less fixed, but not completely - double stars issue still persists
2) Token Selector items styles still don't match figma (different sizes)
3) You can clearly see that there are not margin from a top in Figma (above balance cards), but there is still in app
4) Issue with double state for selected token still unresolved

I've kept all unresolved threads active.

**Comment by whateverfw** [comment]
Issues with select multiplier slider:
1) When 0x is selected there is no progress filled
image

2) With 1x or 3x values progress is not fully filled (there is some space before dot)
image

3) I can't completely select 2x in a slider 

4) Dot is not exactly above multiplier label, it is shifted, e.g. it is not exactly on top of 3x
image

5) Seems like it is not filled with correct color, as in figma (e.g. you can clearly see that progress is blue, when 2x is selected), so progress gradient do not correspond to figma

6) In case of 10x dot should be in the end, as well as 10x label, so no progress on the right should be left
image

**Comment by whateverfw** [comment]
Vertical margins are completely broken and do not correspond to figma:
App:
image
Figma: 
image

**Comment by DanielEmmanuel1** [comment]
Done with majority of the issues
Working on the MultiplierSelector bugs

File: frontend/src/components/MultiplierSelector.jsx
[the remark this answers is not in the document]
Comment (DanielEmmanuel1) [reply frontend/src/components/MultiplierSelector.jsx:1]: > max multiplier value should be dynamic, and depend on what is returned from useMaxMultiplier based on a selected token (currently it is always 10)

The issue progress background of blue been filled when at x0 mark, is not feasible cos with the new math calcs, the x10 mark is no longer been filled, so the approach is to just leave x0 as is.  As for the dynamic approach of the multiplier, i am getting a value of 2.5 from the backend and that wont work correctly to achieve the design.

[the remark this answers is not in the document]
Comment (DanielEmmanuel1) [reply frontend/src/components/MultiplierSelector.jsx:1]: > > max multiplier value should be dynamic, and depend on what is returned from useMaxMultiplier based on a selected token (currently it is always 10)
> 
> The issue progress background of blue been filled when at x0 mark, is not feasible cos with the new math calcs, the x10 mark is no longer been filled, so the approach is to just leave x0 as is. As for the dynamic approach of the multiplier, i am getting a value of 2.5 from the backend and that wont work correctly to achieve the design.

Maybe u can guide me on a better logic to achieve this

[the remark this answers is not in the document]
Comment (whateverfw) [reply frontend/src/components/MultiplierSelector.jsx:1]: @DanielEmmanuel1 let's wait on this, I'll let you know

[the remark this answers is not in the document]
Comment (whateverfw) [reply frontend/src/components/MultiplierSelector.jsx:1]: @DanielEmmanuel1 Okay, you don't need to make more changes. I'm considering that you've finished, thanks.

**Review by whateverfw — APPROVED** [review]
@DanielEmmanuel1 @djeck1432

**Review by djeck1432 — APPROVED** [review]
Looks good to me

**Pull request merged**